src/MicrosoftTeams.ts

```diff
@@ -2329,3 +2329,90 @@ export function getChatMembers(
   const messageId = sendMessageRequest(parentWindow, "getChatMembers");
   callbacks[messageId] = callback;
 }
+
+/**
+ * Namespace to interact with the conversational subEntities inside the tab
+ */
+export namespace conversations {
+
+  /**
+   * @private
+   * Hide from docs.
+   * ------
+   */
+  export interface ConversationalSubEntity {
+
+    /**
+     * A function that is called once the conversation Id has been created.
+     */
+    onStartConversation?: (conversationId: string) => void;
+
+    /**
+     * A function that is called if the pane is closed
+     */
+    onCloseConversation?: (reason: string) => void;
+  }
+
+  /**
+   * @private
+   * Hide from docs
+   * --------------
+   * Allows the user to start a conversation with each subentity inside a tab
+   * @param subEntityId The Id of the subEntity where the conversation is taking place
+   * @param title The title of the conversation
+   * @param conversationalSubEntity Callback containing the conversation Id and if the tab pane was closed
+   */
+  export function startConversation(
+    subEntityId: string,
+    title: string,
+    conversationalSubEntity: ConversationalSubEntity
```

Comment:
It would be better to combine all parameter into one request interface like StartConversationRequest that will help if we add any new parameter then caller don't need to worry about.